src/Uno.UI.Tests/BinderTests/Given_Binder.inpc.cs

```diff
@@ -0,0 +1,225 @@
+
+			Assert.AreEqual(SUT.MyValue, master.Class1.Value);
+			Assert.AreEqual(2, master.Class1.ValueGetCount);
+			Assert.AreEqual(0, master.Class1.ValueSetCount);
+
+			master.Update();
+
+			Assert.AreEqual(SUT.MyValue, master.Class1.Value);
+			Assert.AreEqual(4, master.Class1.ValueGetCount);
+			Assert.AreEqual(1, master.Class1.ValueSetCount);
+		}
+
+		[TestMethod]
+		public void When_INPC_Update_SameReference_Converter()
+		{
+			var SUT = new Binder_INPC_Data();
+			SUT.SetBinding(
+				Binder_INPC_Data.MyValueProperty,
+				new Binding
+				{
+					Path = new PropertyPath("Value"),
+					Converter = new Binder_INPC_DummyConverter()
+				});
+
+			var master = new Binder_INPC_Base_Class();
+			SUT.DataContext = master;
+
+			Assert.AreEqual(2, master.ValueGetCount);
+			Assert.AreEqual(0, master.ValueSetCount);
+
+			master.RaiseUpdated();
+
+			Assert.AreEqual(SUT.MyValue, master.Value);
+			Assert.AreEqual(4, master.ValueGetCount);
+			Assert.AreEqual(0, master.ValueSetCount);
+		}
+
+		[TestMethod]
+		public void When_INPC_Raise_All_Updated()
+		{
+			var SUT = new Binder_INPC_Data();
+			SUT.SetBinding(
+				Binder_INPC_Data.MyValueProperty,
+				new Binding
+				{
+					Path = new PropertyPath("Class1.Value")
+				});
+
+			SUT.SetBinding(
+				Binder_INPC_Data.MyValue2Property,
+				new Binding
+				{
+					Path = new PropertyPath("Value"),
+					Converter = new Binder_INPC_DummyConverter()
+				});
+
+			var master = new Binder_INPC_Base_Class();
+			SUT.DataContext = master;
+
+			master.RaiseAllUpdated();
+
+			Assert.AreEqual(SUT.MyValue, master.Class1.Value);
+			Assert.AreEqual(SUT.MyValue2, master.Value);
+		}
+	}
+
+	public partial class Binder_INPC_Data : DependencyObject
+	{
+		public string MyValuePropertyValueDuringChange { get; private set; }
+
+		public string MyValue
+		{
+			get => GetMyValueValue();
+			set => SetMyValueValue(value);
+		}
+
+		[GeneratedDependencyProperty(DefaultValue = "")]
+		public static DependencyProperty MyValueProperty { get; } = CreateMyValueProperty();
+
+		private void OnMyValueChanged(string oldValue, string newValue)
+		{
+			MyValuePropertyValueDuringChange = MyValue;
+		}
+
+		public string MyValue2
+		{
+			get => GetMyValue2Value();
+			set => SetMyValue2Value(value);
+		}
+
+		[GeneratedDependencyProperty(DefaultValue = "")]
+		public static DependencyProperty MyValue2Property { get; } = CreateMyValue2Property();
+	}
+
+	public class Binder_INPC_Base_Class : Binder_INPC_BaseViewModel
+	{
+		private string _value = Guid.NewGuid().ToString();
+		public int ValueGetCount { get; private set; }
+		public int ValueSetCount { get; private set; }
+
+		public Binder_INPC_Class1 Class1 { get; set; } = new Binder_INPC_Class1();
+		public string Value
+		{
+			get
+			{
+				ValueGetCount++;
+				return _value;
+			}
+			set
+			{
+				ValueSetCount++;
+				_value = value;
+			}
+		}
+
+		public void Update()
+		{
+			Class1.Value = (Binder_INPC_Class1.IndexCounter++).ToString();
```

Comment:
Codacy found an issue: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5775486)